terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/exception/ExceptionLoggingFilter.java

```diff
@@ -159,7 +159,7 @@ protected void logServletException(ServletException ex,
      */
     protected void logRuntimeException(RuntimeException ex,
             ServletRequest request, ServletResponse response) {
-        exceptionLogger.log(ex);
+        exceptionLogger.error(ex);
     }

```

Comment:
@yoshikawaa
It was not good that the discussion was restricted to the migration because my answer was only the answer from the viewpoint of migration.

This issue(#702) concluded that it is not appropriate to use the exception code in logging exceptions handled by the FW.
However, it was not the conclusion that the function of determination the log level by the exception code is unnecessary as the function of the common library.


In the guideline, we also recommend using `ExceptionLogger` within business logic, and I do not want to stop this usage.

I think it is necessary to discuss whether log level determination function by exception code is unnecessary as function of common library.